frontend/src/components/Dashboard/components/LocationItem/LocationItem.tsx

```diff
@@ -0,0 +1,24 @@
+import MemoItem from "../MemoItem/MemoItem";
+
+const LocationItem = (props:{ 
+    locationName:string,
+    expanded:boolean,
+    handleLocationClick:(locationName:string) => void,
+    memos:[],
```

Comment:
nit: add the appropriate type for this and other places where you use memo,

Reply:
*avoid* `any`, might seem like a pita at first, but will make everything smoother as you know for sure the type of an object you're working with and their properties

Reply:
case in point:
```tsx
// line 33, Popup.tsx
...
    selectedMemo.name = newTitle;
    selectedMemo.description = newMemo;
    const updatedMemo = {
      id: selectedMemo**._id**,
      description: selectedMemo.description,
      date: new Date().toLocaleString(),
...
```
- selectedMemo is a `any` type, as you never specify it, so you dont actually know if a property you access actually exists. You're just *assuming* it has it, but it doesn't
- a `MemoType`'s id is `id` not `_id`

Reply:
the above non-existant property also exists several place in the code, not just here

Reply:
the _id vs id is actually a discrepency between the the MemoType interface and mongoose schema. 

The object returned from mongo will have _id but currently the type has a field of id.

A problem I outlined in #64

Reply:
So i believe the refactoring of the types of memo objects to something that isnt any will have to come in a later refactoring PR when the models are made consistent.

Reply:
r u sewious wight neow
(very good catch, forgot about the _id type that mongoose returns)